homeassistant/components/isy994.py

```diff
@@ -132,6 +134,20 @@ def _categorize_programs() -> None:
                         PROGRAMS[component].append(program)
 
 
+def _categorize_weather() -> None:
+    """Categorize the ISY994 weather data."""
+    global WEATHER_NODES
```

Comment:
Yeah, I think I am the only one too. I'll probably just convert my custom components to real components so I wont need them anymore.